docroot/themes/humsci/humsci_colorful/config/install/humsci_colorful.settings.yml

```diff
@@ -3,4 +3,5 @@ lockup:
   option: default
 global_footer_variant_classname: default
 local_footer_variant_classname: default
-theme_color_pairing: ocean
\ No newline at end of file
+theme_color_pairing: ocean
+animation_enhancement: false
```

Comment:
nice! let's ask Alex and Tori today what they want to the default for all sites to be. we can export that as part of another piece of config I believe cc: @kaseybon

Reply:
The JIRA card states that the toggler should be false by default.

I discovered that I don't even need this setting variable. If it doesn't exist then $animation_enhancement is false and the class isn't applied.